<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>444</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>–°–∏—Å—Ç–µ–º–∞ —Å–∫–∞–Ω–∏—Ä–æ–≤–∞–Ω–∏—è —É—è–∑–≤–∏–º–æ—Å—Ç–µ–π</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_scan">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../../.designer/backup/picture/arrow-continue.png</normaloff>../../../../.designer/backup/picture/arrow-continue.png</iconset>
     </attribute>
     <attribute name="title">
      <string>üîç</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_scan_ip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>–°–∫–∞–Ω–∏—Ä–æ–≤–∞–Ω–∏–µ —Å–µ—Ç–∏</string>
      </property>
      <widget class="QPushButton" name="pushButton_scanner_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>–°–∫–∞–Ω–∏—Ä–æ–≤–∞—Ç—å</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_ip_list">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_router_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Default Gateway: -</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_your_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>–í–∞—à IP-–∞–¥—Ä–µ—Å: -</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>IP-–∞–¥—Ä–µ—Å</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_mac_list">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>101</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_name_comp_list">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>121</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>MAC-–∞–¥—Ä–µ—Å</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>–ü—Ä–æ–∏–∑–≤–æ–¥–∏—Ç–µ–ª—å</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_client">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>201</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>–°–∫–∞–Ω–∏—Ä–æ–≤–∞–Ω–∏–µ –ø–æ—Ä—Ç–æ–≤</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser_list_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>181</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_list_port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>–°–∫–∞–Ω–∏—Ä–æ–≤–∞—Ç—å</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ip_port_scan">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>IP-–∞–¥—Ä–µ—Å –¥–ª—è —Å–∫–∞–Ω–∏—Ä–æ–≤–∞–Ω–∏—è –ø–æ—Ä—Ç–æ–≤</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>15</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>255.255.255.255</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_brute">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>121</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>BruteForce</string>
      </property>
      <widget class="QLabel" name="label_ftp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>FTP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ftp">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ssh">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SSH</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_brute">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>101</width>
         <height>131</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ssh">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_brute_ip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>IP-–∞–¥—Ä–µ—Å –¥–ª—è –±—Ä—É—Ç—Ñ–æ—Ä—Å–∞</string>
       </property>
       <property name="maxLength">
        <number>15</number>
       </property>
       <property name="placeholderText">
        <string>255.255.255.255</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_brute_file">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>–ê–¥—Ä–µ—Å —Å–ª–æ–≤–∞—Ä—è –ø–æ –∫–æ—Ç–æ—Ä–æ–º—É –±—É–¥–µ—Ç –ø—Ä–æ–≤–æ–¥–∏—Ç—å—Å—è –±—Ä—É—Ç—Ñ–æ—Ä—Å</string>
       </property>
       <property name="maxLength">
        <number>1000</number>
       </property>
       <property name="placeholderText">
        <string>C:\</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_errors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_errors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>–û–∫–Ω–æ —Å–æ—Å—Ç–æ—è–Ω–∏—è</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_client_ip">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>IP-–∞–¥—Ä–µ—Å –∫–ª–∏–µ–Ω—Ç–∞</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_find_client">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>–ü–æ–∏—Å–∫ –∫–ª–∏–µ–Ω—Ç–∞</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_connect">
     <attribute name="icon">
      <iconset>
       <normaloff>../../../../.designer/backup/picture/blue-folder-open-document.png</normaloff>../../../../.designer/backup/picture/blue-folder-open-document.png</iconset>
     </attribute>
     <attribute name="title">
      <string>üìù</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>–ü—Ä–∏–µ–º —Ñ–∞–π–ª–æ–≤ —Ñ–æ–Ω–æ–º</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser_errors_get">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>271</width>
         <height>51</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_get_file">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>–ü—Ä–∏—ë–º —Ñ–∞–π–ª–æ–≤</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
